spec/recurly/account_spec.rb

```diff
@@ -341,4 +341,19 @@
       account.company_name.must_equal 'My Company Inc.'
     end
   end
+
+  describe "account acquisition" do
+    let(:account) {
+      stub_api_request :get, 'accounts/abcdef1234567890', 'accounts/show-200'
+      stub_api_request :get, 'accounts/abcdef1234567890/acquisition', 'account_acquisition/show-200'
+      Account.find 'abcdef1234567890'
+    }
+
+    it 'is able to retrieve and parse acquisition' do
```

Comment:
I only did this spec because `destroy` seems to be covered in the resource spec, and `save` is already covered above. I can add a method to the `save` describe block if requested